src/org/usfirst/frc/team2335/robot/commands/FeedShooter.java

```diff
@@ -0,0 +1,45 @@
+package org.usfirst.frc.team2335.robot.commands;
+
+import org.usfirst.frc.team2335.robot.Robot;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+
+public class FeedShooter extends Command
+{
+    public FeedShooter() 
+    {
+        requires(Robot.shooter);
+    }
+
+    protected void initialize() 
+    {
+    	
+    }
+
+    protected void execute() 
+    {
+        //Turns on feed motor
+    	System.out.println("Running");
+    	Robot.shooter.feedBall();
+    }
+
+    protected boolean isFinished() 
+    {
+        return Robot.shooter.isShootButtonReleased();
+    }
+
+    protected void end()
+    {
+        //Turns off feed motor and shooter motor
+    	Robot.shooter.stopFeedBall();
+    	Robot.shooter.stopBall();
```

Comment:
It starts the shooter in the previous command that the command group ran, to rev up the shooter motor to full speed before starting the feeding of the balls.